src/vue/components/panel-compact.vue

```diff
@@ -124,59 +137,70 @@
              v-html="data.fullDefinitions"></div>
       </div>
       <div :id="inflectionsPanelID" class="alpheios-panel__tab-panel alpheios-panel__tab__inflections"
-           v-if="data.inflectionComponentData.inflDataReady && data.settings" v-show="inflectionsTabVisible">
-        <inflections :data="data.inflectionComponentData" :inflections-enabled="data.inflectionsEnabled"
-                     :locale="data.settings.locale.currentValue" :wait-state="data.inflectionsWaitState"
-                     @contentwidth="setContentWidth" class="alpheios-panel-inflections">
-        </inflections>
+           v-if="$store.getters[`app/hasInflData`]" v-show="inflectionsTabVisible">
+        <inflections @contentwidth="setContentWidth" class="alpheios-panel-inflections"></inflections>
       </div>
       <div :id="inflectionsBrowserPanelID" class="alpheios-panel__tab-panel alpheios-panel__tab__inflectionsbrowser"
-           v-if="data.inflectionBrowserEnabled && data.settings" v-show="inflectionsBrowserTabVisible">
-        <inflection-browser :data="data.inflectionBrowserData" :infl-browser-tables-collapsed="data.inflBrowserTablesCollapsed"
+           v-if="settings.contentOptions.items" v-show="$store.state.app.tabState.inflectionsbrowser">
+        <inflection-browser :data="data.inflectionBrowserData"
```

Comment:
It definitely should! I did refactor data storage for inflections, but did not have time to do so for an inflection browser. I will adjust the code, it should not be a big change.

Reply:
ok, or as noted above, understand that these are incremental changes, and that's okay but just leave a TODO comment so that we know this still has to be done.

Reply:
I did a little investigation and discovered it is not really used at the moment: we needed it to track visibility changes but it was never updated and the code works well without tracking it. So I went ahead and removed it.